src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/ReferenceDataController.kt

```diff
@@ -19,7 +19,15 @@ class ReferenceDataController(
 
   @GetMapping("/local-delivery-units")
   fun localDeliveryUnits(): List<LocalDeliveryUnitResponse> = LocalDeliveryUnit.values()
-    .map { ldu -> LocalDeliveryUnitResponse(ldu.name, ldu.label) }.toList()
+    .map { LocalDeliveryUnitResponse(it.name, it.label) }.toList()
+
+  @GetMapping("/mappa-levels")
+  fun mappaLevels(): List<MappaLevelResponse> = MappaLevel.values()
+    .map { MappaLevelResponse(it.name, it.label) }.toList()
+
+  @GetMapping("/reasons-for-recalls")
```

Comment:
I would have thought `reasons-for-recall` (no trailing 's') rather than `/reasons-for-recalls`?

Reply:
I thought all endpoints were plural 🤷

Reply:
`/recall-reasons` ?

Reply:
Agreed to change to: `/recall-reasons`